### DISCUSSION TITLE
DBeaver reconnects all editors?!

### DISCUSSION THREAD

**Discussion by peter-petrov** [dbeaver/dbeaver#21203] [General]
I use DBeaver Version 22.1.4.202208051447 

I work via VPN and I am connected to about 3 or 4 PostgreSQL DBs.  

OK, so I keep about 20-30 tabs open normally while I work on VPN. 

Then when I am done, I disconnect from VPN (while I have no queries running), 
and I hibernate my laptop. Then I reconnect again later (say on the next day).  

Now I go to one of my tabs in DBeaver, and I try to run the query that's in it. 

At that point DBeaver decides to reconnect all 20-30 editors (which I have open) before running my query. 
Sometimes I believe I've seen DBeaver reconnect even editors/scripts which I've explicitly closed before. 

**Is there a way to change that behavior and to have DBeaver to just connect that particular editor the query in which I want to run?**

**Comment by LonwoLonwo** [comment]
Hello @peter-petrov 

Could you please provide a video of your issue?
